Sleep Tracker/SleepTracker/Helpers/AppUIConstant.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+
+/// The size of the UI element used in the application.
```

Comment:
According to the opacity, there are not only sizes here. Would it be better this way?
```suggestion
/// The settings of the UI element used in the application.
```

Reply:
Yes. Fixed.